mon-pix/app/components/challenge-item-generic.js

```diff
@@ -15,6 +15,9 @@ export default class ChallengeItemGeneric extends Component {
   @tracked _elapsedTime = null;
 
   _timer = null;
+  _pageBeforeUnloadListener = null;
+  _pageUnloadListener = null;
+  _hasAbandOrValidateAnswer = false;
```

Comment:
```suggestion
  _hasAbandonedOrValidatedAnswer = false;
```
**Suggestion:** Renommage.